README.md

```diff
@@ -15,12 +15,16 @@ Kubeapps is a web-based UI for deploying and managing applications in Kubernetes
 
 Use the Helm chart to install the latest version of Kubeapps: 
 
+***Note:*** Kubeapps supports both Helm v2 and v3. For detailed instructions on how to install and use Kubeapps follow the [Getting Started Guide](docs/user/getting-started.md).
+
 ```bash
 helm repo add bitnami https://charts.bitnami.com/bitnami
+# Helm v2 command
+helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
+# Helm v3 command
+kubectl create namespace kubeapps
 helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
```

Comment:
Looks like a copy-n-pasto, as `--name` isn't supported on the helm3 cli:
```
$ helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
Error: unknown flag: --name
```

Also, we need the `--useHelm3` flag. (EDIT: exactly like you have in the chart readme below)